be/Cf.WebApi/Endpoints/ServiceInformationEndpoints.cs

```diff
@@ -40,7 +40,7 @@ public static void MapServiceInfoRoutes(this IEndpointRouteBuilder builder)
         await additionalInfoService.GetByServiceIdAsync(GetServiceId(httpContextAccessor));
 
     [Authorize(Roles = "Service")]
-    private static async Task<ServiceStatus> GetServiceStatusByIdAsync([FromServices] IServicelnfoService additionalInfoService, IHttpContextAccessor httpContextAccessor) =>
+    private static async Task<Response.ServiceStatusInfo> GetServiceStatusByIdAsync([FromServices] IServicelnfoService additionalInfoService, IHttpContextAccessor httpContextAccessor) =>
```

Comment:
why return object when you can just return string?

Reply:
Because in later updates we might need to add more information with these methods let's say DateUpdated, DateCreated or some other fields we might add. So it is a good practice to have a response object for the method which we can expand.